### PR TITLE
docs: note `nvm install` on setup

### PR DESCRIPTION
**Pull request by Pinjasaur**
## PR Checklist

- [ ] Addresses an existing open issue: fixes #000
- [ ] That issue was marked as [accepting prs](https://github.com/JoshuaKGoldberg/refined-saved-replies/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [ ] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/refined-saved-replies/blob/main/.github/CONTRIBUTING.md) were taken

### CODE REVIEW

File: .github/DEVELOPMENT.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Development
 
-After [forking the repository on GitHub](https://docs.github.com/en/get-started/quickstart/fork-a-repo), install required packages with [Yarn](https://yarnpkg.com):
+After [forking the repository on GitHub](https://docs.github.com/en/get-started/quickstart/fork-a-repo), install required packages with [Yarn](https://yarnpkg.com) and set the correct Node version with [nvm](https://github.com/nvm-sh/nvm#installing-and-updating):
```
Comment (JoshuaKGoldberg) [inline .github/DEVELOPMENT.md:3]: Hmm, I don't love telling everybody that they have to specifically use nvm in order to set up this repository. Is there a phrasing we can use that suggests this without outright requiring it?

Maybe adding below the shell script something like this?

```md
> Tip: Consider using [nvm](https://github.com/nvm-sh/nvm#installing-and-updating) and running `nvm install` to sync the Node version used to develop on this repository.
```

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
A thought! What do you think?

File: .github/DEVELOPMENT.md (hunk 1 above)
Comment (Pinjasaur) [reply .github/DEVELOPMENT.md:3]: I suppose I'm [0/5](https://handbook.mattermost.com/company/about-mattermost/list-of-terms?q=terms#0-5-1-5-2-5-3-5-4-5-5-5) as long as it's documented. I'll make the suggested change.

Comment (JoshuaKGoldberg) [reply .github/DEVELOPMENT.md]: Aside: I like that term a lot... and _really_ like the public docs 😄 very nice.

**Review by JoshuaKGoldberg — APPROVED** [review]
Thanks!

**Pull request merged**